src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.db;
+
+import java.util.Map;
+
+public class Storage {
+    private Map<String, Integer> storage;
+
+    public Storage(Map<String, Integer> map) {
+        this.storage = map;
+    }
+
+    public Map<String, Integer> getStorage() {
+        return storage;
+    }
```

Comment:
I'd recommend you make storage public static final and use it directly in your program. This is just a mock for the database (if we talk about a real program). That avoids you to put storage as a parameter in your services